console/src/main/webapp/manager/app/app.es6

```diff
@@ -177,7 +177,20 @@ angular.module('manager', [
       const alphanum = v => v && v.match(regexp)
       ctrl.$validators.shortname = alphanum
     }
-  }))
+  })).directive('ngConfirmClick', [
+    function () {
+      return {
+        link: function (scope, element, attr) {
+          var msg = attr.ngConfirmClick || 'Are you sure?'
```

Comment:
does it need a translation ? or is it already ?

Reply:
The translation is inside the attribute : `ng-confirm-click="{{ 'org.delete' | translate}} ?"`

In this case it displays : `Delete organization ?`

Reply:
Yes, I saw it, but my question was about the availability of translation strings for 'Are you sure?' and 'Delete organization ?' since they are not provided by the PR.